src/install/apache/whatsNext.mdx

```diff
@@ -0,0 +1,637 @@
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: '150px' }}>
+        Setting
+      </th>
+
+      <th>
+        Description
+      </th>
+
+      <th>
+        Default
+      </th>
+
+      <th style={{ width: '150px' }}>
+        Applies to
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `STATUS_URL`
+      </td>
+
+      <td>
+        The URL set up to provide the metrics using the status module.
+      </td>
+
+      <td>
+        `http://127.0.0.1/server-status?auto`
+      </td>
+
+      <td style={{ "text-align": "center" }}>
+        Metrics, inventory
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        `BINARY_PATH`
+      </td>
+
+      <td>
+        Set location of the Apache binary file.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        Inventory
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        `CA_BUNDLE_FILE`
+      </td>
+
+      <td>
+        Alternative Certificate Authority bundle file.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        Metrics
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        `CA_BUNDLE_DIR`
+      </td>
+
+      <td>
+        Alternative Certificate Authority bundle directory.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        Metrics
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        `VALIDATE_CERTS`
+      </td>
+
+      <td>
+        Set to `false` if the status URL is HTTPS with a self-signed certificate.
+      </td>
+
+      <td>
+        true
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        Metrics
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        [`REMOTE_MONITORING`](/docs/remote-monitoring-host-integrations)
+      </td>
+
+      <td>
+        Enable multi-tenancy monitoring.
+      </td>
+
+      <td>
+        true
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        Metrics, inventory
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        `METRICS`
+      </td>
+
+      <td>
+        Set to `true` to enable metrics-only collection.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}/>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        `INVENTORY`
+      </td>
+
+      <td>
+        Set to `true` to enable inventory-only collection.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}/>
+    </tr>
+  </tbody>
+</table>
+
+## Example configurations [#examples]
+
+Here are some example YAML configurations: 
+
+<CollapserGroup>
```

Comment:
I like how these are organized with collapsers. The descriptions are good, too. I'm not sure what level of knowledge we're assuming here. Is it possible to mix all these example configs together? Are there certain configs that are mutually exclusive? Do we need to describe when they might want to use these configs?

If it's assumed that iykyk, then disregard the comment. just wanted to call out it wasn't clear to me